src/rofl/common/crofbase.h

```diff
@@ -1169,9 +1169,9 @@ class crofbase : public virtual rofl::cthread_env,
    * @param auxid control connection identifier
    * @param msg OpenFlow message instance
    */
-  virtual void
-  handle_get_config_reply(rofl::crofdpt &dpt, const rofl::cauxid &auxid,
```

Comment:
indeed, but I even shouldn't have done these drive by fixes here. I will add the rest in an additional PR.